recipe/meta.yaml

```diff
@@ -33,7 +33,7 @@ build:
 
 requirements:
   build:
-    - python  # [win]
+    - python 2.7* # [osx or win]
```

Comment:
The Windows part has to do with `vc` features. So we can't pin it under the Windows selector. Though we could pin it under an `osx ` only selector.